arbor/fvm_lowered_cell.hpp

```diff
@@ -211,6 +211,13 @@ struct fvm_lowered_cell {
         std::vector<deliverable_event> staged_events,
         std::vector<sample_event> staged_samples) = 0;
 
+    virtual cell_labeled_ranges detector_data() const = 0;
+    virtual cell_labeled_ranges synapse_data() const = 0;
+    virtual cell_labeled_ranges gap_junction_data() const = 0;
+
+    virtual fvm_size_type num_detectors(cell_gid_type gid) const = 0;
+    virtual fvm_size_type num_synapses(cell_gid_type gid) const = 0;
+
```

Comment:
I guess you mean `mc_cell_group` instead of `mc_cable_cell`?
I would agree, but we actually need the `cell_labelled_ranges` in `fvm_lowered_cell_impl` in order to resolve the gap junction labels.

Reply:
I did indeed mean `mc_cell_group`, thanks :)
That resolution though is required only during the initialization phase; there's no need to hold on to it down in `fvm_lowered_cell_impl` after that, right?